openmrs/omod/src/main/java/org/projectbuendia/openmrs/web/controller/ProfileManager.java

```diff
@@ -48,34 +42,70 @@
 /** The controller for the profile management page. */
 @Controller
 public class ProfileManager {
-    static Log log = LogFactory.getLog(ProfileManager.class);
-    final File PROFILE_DIR = new File("/usr/share/buendia/profiles");
-    final String VALIDATE_CMD = "buendia-profile-validate";
-    final String APPLY_CMD = "buendia-profile-apply";
+    protected static Log log = LogFactory.getLog(ProfileManager.class);
+    private final String PROFILE_DIR_PATH = "/usr/share/buendia/profiles";
+    private final String VALIDATE_CMD = "buendia-profile-validate";
+    private final String APPLY_CMD = "buendia-profile-apply";
+    private File profileDir;
+
+    @ModelAttribute
+    public void getInitializedMyObject() {
+        Properties config = Context.getRuntimeProperties();
+        String configProfileDir = config.getProperty("profile_manager.profile_dir");
```

Comment:
`getProperty()` has a nice feature that it takes a default value as a second argument, so this can be simplified to:

`config.getProperty("profile_manager.profile_dir", PROFILE_DIR_PATH)`